Add `selected_resources` Jinja context to docs - v1.1 (#1333)
fishtown-analytics/docs.getdbt.com: website/docs/reference/dbt-jinja-functions/selected_resources.md

---
title: "selected_resources"
id: "selected_resources"
---

The `selected_resources` context variable contains a list of all the _nodes_ 
selected by the current dbt command. 

Currently, this variable is not accessible when using the command `run-operation`.

> :::danger Heads up

Reviewer: ```suggestion
:::danger Warning!
```

dbt actively builds the graph during the [parsing phase](execute) of
running dbt projects, so the `selected_resources` context variable will be

> empty during parsing. Please read the information below carefully

Reviewer: ```suggestion
empty during parsing. Please read the information on this page to effectively use this variable.
```

> to understand how to effectively use this variable.

Reviewer: ```suggestion
```

---
title: "selected_resources"
id: "selected_resources"
---

The `selected_resources` context variable contains a list of all the _nodes_ 
selected by the current dbt command. 

Currently, this variable is not accessible when using the command `run-operation`.

:::danger Heads up

dbt actively builds the graph during the [parsing phase](execute) of
running dbt projects, so the `selected_resources` context variable will be
empty during parsing. Please read the information below carefully
to understand how to effectively use this variable.

> :::

Reviewer 2: 🙏 thank you
